docs/content/en/release-notes/_index.md

```diff
@@ -57,9 +57,168 @@ The K8ssandra helm chart deploys the following components. Some are optional, an
+{{% /alert %}}
+
+### Upgrading to K8ssandra 1.3.0 and Cassandra 4.0
+
+When you upgrade from a prior K8ssandra release to 1.3.0, the default `cassandra.version` is now `4.0.0`. A changed `num_tokens` default may prevent Cassandra 4.0 from starting. This section explains how to avoid the issue. 
+
+First, some background information. The default value for `num_tokens` in Cassandra 3.11 is 256. The default in Cassandra 4.0 is 16. The `num_tokens` setting defines the number of tokens randomly assigned to this node on the ring. The more tokens, relative to other nodes, the larger the proportion of data that this node will store.
+
+The issue: [Upgrading from Cassandra 3.11 to 4.0 fails if num_tokens is not set](https://github.com/k8ssandra/k8ssandra/issues/1029). As noted for this scenario, the Cassandra logs will include the following error, and Cassandra will not start:
+
+```
+org.apache.cassandra.exceptions.ConfigurationException: Cannot change the number of tokens from 256 to 16
+```
+
+To avoid the issue, first check the `num_tokens` settings in your K8ssandra-deployed Cassandra 3.11 database. Let's walk through an example.
+
+#### Initial installation
+
+This example starts by installing K8ssandra 1.3.0 with Cassandra 3.11.10 and the latter's default `num_tokens` (256). 
+
+```bash
+helm install k8ssandra -f k8ssandra.values.yml k8ssandra/k8ssandra
+```
+
+k8ssandra.values.yml:
+
+```yaml
+cassandra:
+  version: "3.11.10"
+  datacenters:
+    - 
+      name: dc1
+      size: 1
+stargate:
+  enabled: false
+```
+
+Notice that `stargate` is disabled here just for the sake of quickly starting and testing the full deployment.
+
+**Resulting deployment:**
+
+```bash
+helm list
+```
+**Output**:
+```
+NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART           APP VERSION
+k8ssandra       default         1               2021-08-17 18:58:03.831212 -0400 EDT    deployed        k8ssandra-1.3.0 
+```
+
+After about 10 minutes, verify the Cassandra Operator status:
+
+```bash
+kubectl describe cassdc dc1 | grep "Cassandra Operator Progress:"
+```
+**Output**:
+```
+  Cassandra Operator Progress:  Ready
+```
+
+Verify the server version:
+
+```bash
+kubectl describe cassdc dc1 | grep "Server Version:" 
+```
+**Output**:
+```
+  Server Version:  3.11.10
+```
+
+Verify the `num_tokens`, in this case defined by default value:
+
+```bash
+kubectl describe cassdc dc1 | grep "num_tokens:"
+```
+**Output**:
+```
+      num_tokens:                         256
+```
+
+#### Upgrades
+
+Upgrade to Cassandra 4.0.0 and explicitly sets the `num_tokens` to match the previous default (256).
+
+```bash
+helm upgrade k8ssandra -f k8ssandra-upgrade.values.yml k8ssandra/k8ssandra
+```
+
+k8ssandra-upgrade.values.yml:
+
+```yaml
+cassandra:
+  version: "4.0.0"
+  datacenters:
+    - 
+      name: dc1
+      size: 1
+      num_tokens: 256
+stargate:
```

Comment:
Same comment here, we can simplify a bit just by taking this out of the example.

Reply:
ok